geonode/messaging/models.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# ########################################################################
+#
+# Copyright (C) 2020 OSGeo
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# ########################################################################
+
+# Its required to fire post_migrate
```

Comment:
@petrus7, what do you mean by `Its required to fire post_migrate`?

Reply:
This is required to trigger post_migrate signal. That signal is not fired if models.py does not exist in the app